### PR TITLE
Add version check on Elyra libraries

### PR DESCRIPTION
**Pull request by lugi0**

### CODE REVIEW

**Review by bdattoma — DISMISSED** [review]

File: ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot [hunk 1]
```diff
@@ -385,6 +385,12 @@ Verify Installed Library Version
     IF  '${status}' == 'FAIL'  Run Keyword And Continue On Failure  FAIL  "Expected ${lib} at version ${ver}, but ${value}"
     RETURN    ${status}    ${value}
 
+Verify Installed Labextension Version
+    [Arguments]  ${lib}  ${ver}
+    ${status}  ${value} =  Run Keyword And Warn On Failure  Run Cell And Check Output  !jupyter labextension list 2>&1 | grep ${lib} | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"v"); printf "%s", b[2]}'  ${ver}  #robocop: disable
```
Comment (jstourac) [inline ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot:390]: Something like

```
!jupyter labextension list 2>&1 | grep ${lib} | tr -s ' ' | cut -d ' ' -f 3 | tr -d 'v'
```

looks more readable for me, but that is just a matter of point of view. Keep it the way you have it now.

**Review by jstourac — DISMISSED** [review]
LGTM, once a verified successful run exist, it's good to go from my point of view :slightly_smiling_face:

**Comment by lugi0** [comment]
Verified in rhods-sanity/1545

File: ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot (hunk 1 above)
Comment (lugi0) [reply ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot]: I'm personally more familiar with the `awk` syntax, we can open it up for discussion with other folks and see what everyone thinks

**Review by jstourac — APPROVED** [review]

**Pull request merged**